README.md

```diff
@@ -0,0 +1,60 @@
+# Lunie Official Monorepo
+
+## Packages:
+
+- API: an API to interact with many different PoS blockchains
+
+- App: Lunie Core (webapp)
+
+- Extension: Lunie browser extension.
+
+## Preparation:
+
+To install the required modules for all packages run:
+
+```bash
+$ yarn workspace api
+$ yarn workspace app
+$ yarn workspace extension
+```
+
+Or just work on the directory you are interested in and simply run `yarn`
+
+#### To install a single package:
+
+Should follow this syntax:
+
+```bash
+$ yarn workspace <workspace-name> --dev <package-name>
```

Comment:
```suggestion
$ yarn workspace <workspace-name> <package-name>
```

`--dev` only adds dev dependencies, right?

Reply:
yes... yeah, should be included in the improved version